### PR TITLE
chore: update context/options

### PR DESCRIPTION
**Pull request by Meng-Heng**
Is this enough information, @darcywong00?

### CODE REVIEW

**Comment by darcywong00** [comment]
Follow-on to #1148 in updating Developer context content

File: developer/17.0/context/options.md [hunk 1]
```diff
@@ -121,7 +121,17 @@ Update database
 ![Options dialog - Server tab](/cdn/dev/img/developer/170/ui/frmOptions_Server.png)
 
 Keyman Developer Server
-: *update soon*
+:   ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmKeyman_Developer_Server_Options.png)   
+    
+    Testing out custom keyboards on a server with the new configuration of Keyman Develper Server will only enhanced the usability of the keyboard, with many features including:
```
Comment (darcywong00) [inline developer/17.0/context/options.md:126]: Good update. I suggest rephrasing like:

```suggestion
    You can customize the Keyman Developer server for testing the usability of your keyboard. Features of the Keyman Developer server include:
```

File: developer/17.0/context/options.md [hunk 2]
```diff
@@ -121,7 +121,17 @@ Update database
...
+    * Allow testing cross devices
+    * Wider range support for NGROK
+    
+    [Find out more](https://github.com/keymanapp/keyman/pull/6073)
 
 List local URLs for Server
-: *update soon*
\ No newline at end of file
+:   Untick the box to close showing any local URLs for keyboard testing
+    ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmList_Local_URLs.png)
```
Comment (darcywong00) [inline developer/17.0/context/options.md:137]: I'm unclear where this section is from the `frmKeyman_Developer_Server_Options.png` screenshot.

Maybe you can add more content about the tickboxes for:

Leave server running after closing IDE
Use ngrok to provide public url for web debugger
Show server console window on start

Comment (Meng-Heng) [reply developer/17.0/context/options.md:159]: I added the full length of information for the instructions above, Darcy. This section is related to another configuration of Keyman Developer Server here:
image

File: developer/17.0/context/options.md [hunk 3]
```diff
@@ -156,4 +156,6 @@ Configure Server...
 
 List local URLs for Server
 :   Below Configure Server, untick the box, and it will quit displaying any local URLs for keyboard testing.
-    ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmList_Local_URLs.png) 
\ No newline at end of file
+    ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmList_Local_URLs.png) 
+
+Explore more: [Keyman Developer Server](http://localhost:8055/developer/17.0/context/server#toc-configuring-keyman-developer-server)
```
Comment (darcywong00) [inline developer/17.0/context/options.md:161]: Can we make this a relative path? See if this works
```suggestion
Explore more: [Keyman Developer Server](server#toc-configuring-keyman-developer-server)
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**